_layouts/category_index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+---
+{% if page.paginator %}
+
+  <!-- Pagination is active -->
+  {% assign paginator = page.paginator %}
+
+  {% if paginator.previous_page != nil or paginator.next_page != nil %}
+    <!-- Show page number if we have multiple pages -->
+    <h1>Category index page {{ paginator.page }} / {{ paginator.total_pages }} for &ldquo;{{ page.title }}&rdquo;</h1>
+  {% else %}
+    <!-- No page number if we only have a single page -->
+    <h1>Category index for &ldquo;{{ page.title }}&rdquo;</h1>
+  {% endif %}
+
+  <a class="tla-link" href="{{site.baseurl}}/blog/category/">Click me to go to a list of categories!</a>
```

Comment:
While this functionality is definitely needed, it might be better off above the title of the page:

![image](https://user-images.githubusercontent.com/7704542/99060906-99efa780-2555-11eb-9e2e-12fcd3f4c727.png)

Roping in @malsf21 for more input on this front.